src/components/appointmentsTable/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Container } from '@mui/material';
+import dayjs, { Dayjs } from 'dayjs';
+import { useState } from 'react';
+import DatePick from './datePicker';
+import AppointmentsTable from './table';
+import { TContainer } from './types';
+
+const AppointmentTableContainer = ({ id }: TContainer) => {
+  const [date, setDate] = useState<Dayjs | string >(dayjs('2023-01-1'));
+  const [loading, setLoading] = useState<boolean>(true);

```

Comment:
why not to move `loading` to `AppointmentsTable` as state it's not sharable at the parent component